### PR TITLE
fix: change frontconf name

### PR DESCRIPTION
**Pull request by oranges-eating**
Signed-off-by: killer <1533063601@qq.com>

### CODE REVIEW

**Comment by oranges-eating** [comment]
demo:http://test.casbin.com:7000/admin/frontconf

File: web/src/locales/zh/data.json [hunk 1]
```diff
@@ -692,14 +692,14 @@
...
-    "forumName": "论坛名字",
-    "logoImage": "logo图片",
-    "footerLogoImage": "底部logo图片",
-    "footerLogoUrl": "底部logo图片超链接",
-    "signinBoxStrongText": "登录栏标题",
-    "signinBoxSpanText": "登录栏子标题",
-    "footerDeclaration": "底部描述",
-    "footerAdvise": "底部建言",
+    "Forum name": "论坛名字",
+    "Logo image": "logo图片",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json:696]: Logo URL

File: web/src/locales/zh/data.json [hunk 2]
```diff
@@ -692,14 +692,14 @@
...
+    "Footer-logo image": "底部logo图片",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json]: Footer Logo URL

File: web/src/locales/zh/data.json [hunk 3]
```diff
@@ -692,14 +692,14 @@
...
+    "Footer-logo Url": "底部logo图片超链接",
+    "Signinbox StrongText": "登录栏标题",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json]: Right title

File: web/src/locales/zh/data.json [hunk 4]
```diff
@@ -692,14 +692,14 @@
...
+    "Signinbox SpanText": "登录栏子标题",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json]: Right subtitle

File: web/src/locales/zh/data.json [hunk 5]
```diff
@@ -692,14 +692,14 @@
...
+    "Footer declaration": "底部描述",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json]: Footer title

File: web/src/locales/zh/data.json [hunk 6]
```diff
@@ -692,14 +692,14 @@
...
+    "Footer advise": "底部建言",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json]: Footer subtitle

File: web/src/locales/zh/data.json (hunk 1 above)
Comment (hsluoyz) [inline web/src/locales/zh/data.json:696]: Logo URL

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Pull request merged**